tests/controllers/user.test.js

```diff
@@ -258,10 +258,16 @@ test("user unsubscribe GET request with valid token and hash returns 200 without
   // Call code-under-test
   await user.getUnsubscribe(req, resp);
 
-  expect(resp.statusCode).toEqual(200);
+  expect(resp.statusCode).toEqual(302);
+  expect(resp._getRedirectUrl()).toEqual("/user/preferences");
 });
 
 
+// TODO: test("remove-email POST");
+// TODO: test("remove-fxm get");
+// TODO: test("remove-fxm POST");
```

Comment:
Are you doing these tests in _this_ PR, or is this work for Future Luke(tm)?

Reply:
Planning to add in _this_ PR.

Reply:
![](https://media.giphy.com/media/l3V0sNZ0NGomeurCM/giphy.gif)

Reply:
Now with tests.